### PR TITLE
[ADD] oauth2 security resolve #8

### LINKED ISSUE #8: Add oauth2 to server

### PR DESCRIPTION
**Pull request by kostya05983**

### CODE REVIEW

File: api/src/main/kotlin/bomber/security/TokenAuthenticationFilter.kt [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+        } catch (ex: Exception) {
+            logger.error("Could not set user authentication in security context", ex)
+        }
+
+        filterChain.doFilter(request, response)
+    }
+
+    private fun getJwtFromRequest(request: HttpServletRequest): String? {
+        val bearerToken = request.getHeader("Authorization")
+        return if (StringUtils.hasText(bearerToken) && bearerToken.startsWith("Bearer ")) {
```
Comment (kubitre) [inline api/src/main/kotlin/bomber/security/TokenAuthenticationFilter.kt:47]: what will happen if the token looks like this:
```
Bearer  
```

Comment (kubitre) [reply api/src/main/kotlin/bomber/security/TokenAuthenticationFilter.kt:47]: after key word Bearer 2 space syms

Comment (kostya05983) [reply api/src/main/kotlin/bomber/security/TokenAuthenticationFilter.kt:47]: true

Comment (kostya05983) [reply api/src/main/kotlin/bomber/security/TokenAuthenticationFilter.kt:47]: I fix it with

Comment (kostya05983) [reply api/src/main/kotlin/bomber/security/TokenAuthenticationFilter.kt:47]: I can made a substring after last space after bearer.

Comment (kostya05983) [reply api/src/main/kotlin/bomber/security/TokenAuthenticationFilter.kt:47]: Fixed it

**Review by kubitre — APPROVED** [review]
ok

**Pull request merged**